### PR TITLE
Add support for rails 5.1:

### PR DESCRIPTION
**Pull request by Edouard-chin**
Hey 👋 ,

Thanks for creating this project!

- Small PR to allow this gem to be included in rails 5.1 applications
- Added a travis matrix for testing both 5.0 and 5.1

cc\ @rafaelfranca

### CODE REVIEW

File: partially_useful.gemspec [hunk 1]
```diff
@@ -18,8 +18,9 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'railties', '>= 4.1', '< 5.1'
+  spec.add_dependency 'railties', '>= 4.1', '< 5.2'
```
Comment (rafaelfranca) [inline partially_useful.gemspec]: Maybe just `'>= 4.1'`?

File: partially_useful.gemspec [hunk 2]
```diff
@@ -18,8 +18,9 @@ Gem::Specification.new do |spec|
...
 
   spec.add_development_dependency "bundler", "~> 1.6"
   spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "responders", "~> 2.0"
```
Comment (rafaelfranca) [inline partially_useful.gemspec]: I think you can't specify the  version otherwise it will not work with rails 4

Comment (Edouard-chin) [inline partially_useful.gemspec]: Good catch, thanks 🥇

**Comment by phoet** [comment]
great, thanks! looks like this is still used at shopify, i likeeey 😸

**Pull request merged**